src/admin/seeds/dashboard/dashboard.html

```diff
@@ -57,7 +57,7 @@
               <div class="featureDescription">
                 Add an account to the whitelist
               </div>
-              <div>
+              <div class='featureContainer__inputWrapper'>
```

Comment:
Can we be consistent with the class naming here and just call this "inputWrapper"?

Reply:
due to time constraints, fine. :)
for the record though, this is a correct naming approach - it's tightly couple with that element and context, namespace prevents leaking and it's explicit that it refers to this one element, since `inputWrapper` is quite likely to be used for something else, elsewhere.

Reply:
understood.  the reason I am requesting consistency is to avoid needlessly adding another pattern and thus complexity and risk to the app.